sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/CosmosConfigSpec.scala

```diff
@@ -27,7 +27,7 @@ class CosmosConfigSpec extends UnitSpec {
     endpointConfig.applicationName.get shouldEqual "myapp"
     endpointConfig.useGatewayMode shouldEqual true
     endpointConfig.preferredRegionsList.isDefined shouldEqual true
-    endpointConfig.preferredRegionsList.get should contain theSameElementsAs Array("westus", "eastus1")
+    endpointConfig.preferredRegionsList.get should contain theSameElementsAs Array("west us", "eastus1")
```

Comment:
I think having a test case for one of the Regions with two variations - like East US2 would be good - like should be remove any whitespace between letters and digits?

Any implementation is fine by me - just want this to be covered in unit test - and please also update the description in the preferredregions config entry (in the config reference md) so that it is clear which format to use.